src/Hooks/useDiningData.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios'
+import {useQuery} from 'react-query'
+async function FetchData(){
+    return await axios.get('http://localhost:4001/Dining');
+}
+export default function useDiningData() {
```

Comment:
we can use a common hook here, we have to only change url. Pass url as a parameter , call useQuery on it directly. You need not create a FetchData for every url. Boil this down to a single function with a paramter